source/load-balancers/tests/server-selection.yml

```diff
@@ -0,0 +1,49 @@
+description: server selection for load-balanced clusters
+
+schemaVersion: '1.3'
+
+runOnRequirements:
+  - loadBalanced: true
+
+createEntities:
+  - client:
```

Comment:
It's an expectation that the cluster URI will include `loadBalanced=true` because this is needed for the internal client as well as all client entities. The only time an explicit `loadBalanced` URI option is needed is if it's intentionally being set to an incorrect value (e.g. being set to true for a cluster that's not behind an LB) to test error cases.